### DISCUSSION TITLE
请问后续有计划完善视频合集与订阅功能吗？

### DISCUSSION THREAD

**Discussion by zcq15** [ywmoyue/biliuwp-lite#293] [General]
当前的视频合集与订阅相关功能不是很完善，在播放视频合集时体验较差，包括：
1.“点击头像-我的收藏-我的收藏与订阅”列表不完整，只显示前19个订阅合集
2.进入up主个人空间，无法查看up主创建的合集列表
3.在“点击头像-我的收藏-我的收藏与订阅”中打开视频合集，右上角有收藏按钮，但是点击之后只在收藏与订阅中创建了一个空的收藏夹，没有任何内容
请问一下后续是否有完善合集与订阅相关功能的计划？

**Comment by ywmoyue** [comment]
有空就搞
